packages/utils/src/browser/hello.ts

```diff
@@ -32,7 +32,7 @@ export function sayHello(type: string): void
     if (navigator.userAgent.toLowerCase().indexOf('chrome') > -1)
     {
         const args = [
-            `\n %c %c %c PixiJS ${VERSION} - ✰ ${type} ✰  %c  %c  http://www.pixijs.com/  %c %c ♥%c♥%c♥ \n\n`,
+            `\n %c %c %c PixiJS ${VERSION} - ✰ ${type} ✰  %c  %c  https://pixijs.com/  %c %c ♥%c♥%c♥ \n\n`,
```

Comment:
https://pixijs.com doesn't load without the "www." domain in front. Maybe a DNS redirect is needed?

Reply:
Really gotta fix that DNS. I'll revert this bit.